scripts/test_aktualizr.sh

```diff
@@ -0,0 +1,15 @@
+#! /bin/bash
+set -e
+
+dpkg-deb -I /persistent/aktualizr.deb && dpkg -i /persistent/aktualizr.deb
+aktualizr --version | grep $(cat /persistent/aktualizr-version) && aktualizr-info
+
+PLAY_SESSION=eyJhbGciOiJIUzI1NiJ9.eyJkYXRhIjp7Im5hbWVzcGFjZSI6ImF1dGgwfDVhN2VmNDU1OTZiMjBjNzUwMjg2ZGRlNSIsImlkX3Rva2VuIjoiZXlKMGVYQWlPaUpLVjFRaUxDSmhiR2NpT2lKU1V6STFOaUlzSW10cFpDSTZJbEV3V2tKU1JFVjNUbnBPUms5VVozaFJhbFY2VGtSQmQwMVVhM2RPUkZGM1QxUktSRTVVVFhwUlZWVjNUVlJTUjA1RVVrZFBVU0o5LmV5SnVhV05yYm1GdFpTSTZJbk5sY21ocGVTNXpkR1YwYzJ0dmRubGphQ3R6Wld4bWRYQmtZWFJsSWl3aWJtRnRaU0k2SW5ObGNtaHBlUzV6ZEdWMGMydHZkbmxqYUN0elpXeG1kWEJrWVhSbFFHbHVibTkwWld0aExtTnZiU0lzSW5CcFkzUjFjbVVpT2lKb2RIUndjem92TDNNdVozSmhkbUYwWVhJdVkyOXRMMkYyWVhSaGNpOWlZVGd4TURBNFkyTTRZV0ZpWTJWak56VTRZV0ZqTXpFd05EY3daamMzTWo5elBUUTRNQ1p5UFhCbkptUTlhSFIwY0hNbE0wRWxNa1lsTWtaalpHNHVZWFYwYURBdVkyOXRKVEpHWVhaaGRHRnljeVV5Um5ObExuQnVaeUlzSW5Wd1pHRjBaV1JmWVhRaU9pSXlNREU0TFRBeUxURXhWREl6T2pJNE9qRXdMams0TTFvaUxDSmxiV0ZwYkNJNkluTmxjbWhwZVM1emRHVjBjMnR2ZG5samFDdHpaV3htZFhCa1lYUmxRR2x1Ym05MFpXdGhMbU52YlNJc0ltVnRZV2xzWDNabGNtbG1hV1ZrSWpwMGNuVmxMQ0pwYzNNaU9pSm9kSFJ3Y3pvdkwyRjBjeTVsZFM1aGRYUm9NQzVqYjIwdklpd2ljM1ZpSWpvaVlYVjBhREI4TldFM1pXWTBOVFU1Tm1JeU1HTTNOVEF5T0Raa1pHVTFJaXdpWVhWa0lqb2lSMHRFWlVKWGFrMUxia0poYURGcVExSk9NbEJ3UTJsSlJFSm9jSGd5Tm1FaUxDSnBZWFFpT2pFMU1UZ3pPVEUyT1RFc0ltVjRjQ0k2TVRVeE9EUXlOelk1TVgwLlV3TlYwUnJDeDljZW1PclM1cFFrMGhELTE5VWNqM3lwLXp1RmNBb082aDFLdmpEdktmX19US0FQT3VmTFU4Wnd5OWtBY21NQk5hQ0laVVA1dVlXVUktb04xRTU0Zkw1cDB6M0gyMTRWWDRDMTlxRUlKTXBmWGM1RW5XTjFKQUItOEpZcmRDZXJFcUJoRnJTdnhBR3hCV2hxaXg5MWg5T0xxS1VLdXpydXR4TVpBWWQ2WW9nODN4ZkFzM3oxQnpOaS1pZDlUN2RSRFZuN21qeUlXVkUtS2djWUVvdGRKMkJrdGZzV3p0NGtOT1BNREVnbjBiN0Q2Zzl3a0RFSWQ3S3JvTG52SjE5RzNranJ4cnY4VnBDWVdtTHhybGxqU2YtQWNOLTNKczA2ZGNhZF84Yk5pYmFXYUFLYjMtYk9SUXVZbjdUUEh2RURPaHU4MEc3UWZReDJlQSIsImFjY2Vzc190b2tlbiI6IntcImFjY2Vzc190b2tlblwiOlwiZXlKaGJHY2lPaUpJVXpJMU5pSXNJbXRwWkNJNmJuVnNiSDAuZXlKcGMzTWlPaUpvZEhSd2N6b3ZMMkYxZEdndGNHeDFjeTVoWkhaaGJtTmxaSFJsYkdWdFlYUnBZeTVqYjIwaUxDSmhkV1FpT2xzaWRHOXJaVzRpTENKamJHbGxiblFpWFN3aWFuUnBJam9pVUd4c05HeDJiRFpZV2taU1VFbzJOVWhVWjNnaUxDSmpiR2xsYm5SZmFXUWlPaUl4TldJd01XUmlaaTFtTjJWbUxUUXpZV1F0WVdaaE1pMWhZamczWW1abE5EQmpZV0VpTENKemRXSWlPaUpoZFhSb01IdzFZVGRsWmpRMU5UazJZakl3WXpjMU1ESTRObVJrWlRVaUxDSnZkMjVsY2lJNmJuVnNiQ3dpYVdGMElqb3hOVEU0TXpreE5qa3hMQ0psZUhBaU9qRTFNVGcwTnpnd09URXNJbk5qYjNCbElqb2lkRzlyWlc0dWFXNTBjbTl6Y0dWamRDQmpiR2xsYm5RdWNtVm5hWE4wWlhJZ1kyeHBaVzUwTG5Wd1pHRjBaU0o5LmJPb3NXUHo4VV9iNTJlMHByRmZCRTJDN3l0eDBHNkxKWllhbkJ0ZFo2Z3NcIixcIm5vdEFmdGVyXCI6MTUxODQ3ODA5MH0iLCJjc3JmVG9rZW4iOiJjN2FlMjA1MTRkN2Y1ZTc5MWE3ZTM5YTc2MzcyMGQyMTIyNGM4ZDdjLTE1MTgzOTE2OTE3NjEtZDA2MGYxNjE3NWRhMWJiZWY5MGUzYTM4In0sIm5iZiI6MTUxODM5MTY5MSwiaWF0IjoxNTE4MzkxNjkxfQ.9LaqhbNXZ5HgCmMdWmilOSETur0Ec4ROvlUkS2Zii8w
```

Comment:
I don't think we can hard-code a session-cookie in here, for two reasons:

1) The Aktualizr repo is public, and I'm a bit concerned about letting anyone who can scrape github have access to one of our accounts.  I can't see what they would do with it right now, but it is certainly bad security hygiene.

2) The cookie will expire, and the tests will stop working until someone fixes the test up, and I think this will happen rather regularly.

Would it be better to take this in via a cmake variable?  This would mean that people could set it locally to use their own accounts, and there wouldn't be problems of it expiring, and people can access the account for debugging?

Reply:
I knew about 2. And didn't think about 1. You are right this is security issue.
But we could use travis variable settings to setup login and password for test account, then we should in script somehow retrive a PLAY_SESSION